packages/gallery/src/components/gallery/proGallery/slideshowView.js

```diff
@@ -446,22 +458,18 @@ class SlideshowView extends GalleryComponent {
 
   handleSlideshowKeyPress(e) {
     e.stopPropagation();
-    switch (e.charCode || e.keyCode) {
-      case 38: //up
-      case 37: //left
-      case 33: //page up
-        e.preventDefault();
-        this._next({ direction: -1, isKeyboardNavigation: true });
-        return false;
-      case 39: //right
-      case 40: //down
-      case 32: //space
-      case 34: //page down
-        e.preventDefault();
-        this._next({ direction: 1, isKeyboardNavigation: true });
-        return false;
+    const nextKeys = [32, 33, 34, 37, 38, 39, 40]
```

Comment:
please keep the comments on char code vs key for readability 
like so:
[ 32, // ?
  33, // page up
.....
]